setup/README.md

```diff
@@ -0,0 +1,63 @@
+# Apigee Platform API Samples Setup
+
+This directory provides scripts that configure your local environment and deploy sample API proxies to your organization on the Apigee API Platform.
```

Comment:
this file hasn't been fromatted?

Reply:
Ah sorry these READMEs and scripts were taken from the apigee/api-platform-samples repository and I left them as is. I'll format them now